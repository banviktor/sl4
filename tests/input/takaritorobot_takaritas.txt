randomization off
questions off
start_game 2
2 1
8 9
toggle_oil
inputvector 0 1
jump
list_smudges
jump
I
3 1
inputvector 0 -1
jump
list_smudges
jump
N
list_smudges
exit
