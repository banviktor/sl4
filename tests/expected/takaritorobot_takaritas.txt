Véletlenszerû események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Olajfolt be.
Ragacs ki.
Inputvektor: 0.0, 1.0
Olajfolt lehelyezve.
Új pozíció: 2.0, 2.0
Foltok listája:
	Olaj: 2.0, 1.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Új pozíció: 8.0, 9.0
Takarítórobot indul útnak innen: 3.0, 1.0
Ez a(z) 2. kör.
A Piros Robot következik.
Inputvektor: 0.0, -1.0
Új pozíció: 2.0, 2.0
Foltok listája:
	Olaj: 2.0, 1.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Új pozíció: 8.0, 9.0
Ez a(z) 3. kör.
A Piros Robot következik.
Foltok listája:
